### ISSUE TITLE
Installation of ruby-1.9.3-p0 failed!

### ISSUE THREAD

**Issue by fny** [remear/jewelrybox#56]
Compiling Ruby 1.9.3 requires a non-LLVM version of GCC, which is no longer included with XCode 4.2:

```
configure: WARNING: unrecognized options: --with-libyaml-dir
checking build system type... x86_64-apple-darwin11.2.0
checking host system type... x86_64-apple-darwin11.2.0
checking target system type... x86_64-apple-darwin11.2.0
checking whether the C compiler works... no
configure: error: in `/Users/farazyashar/.rvm/src/ruby-1.9.3-p0':
configure: error: C compiler cannot create executables
```

While one could install GCC or downgrade to XCode, a simpler solution is to configure the installation to use clang, a C-family frontend for LLVM, as the compiler:

```
rvm install 1.9.3 --with-gcc=clang
```

clang's inclusion began XCode 3.2 (2009?), so perhaps one could include the argument by default.

**Comment by remear** [comment]
--with-gcc=clang does not work on all machines and for all gems installed for such rubies. It's more of a use at your own risk option. No optimal solution has been found so we've held off on adding an option into JewelryBox. I think we will add one in the next release but that will pretty much be the extend of the support for now.

**Comment by remear** [comment]
Use Clang option added to Ruby installation options and will e available in JewelryBox v1.2.
